module/augment/conversion_unique.py

```diff
@@ -0,0 +1,321 @@
+    def handle_convt2_finish(self, skip_first_screenshot=True):
+        while 1:
+            if skip_first_screenshot:
+                skip_first_screenshot = False
+            else:
+                self.device.screenshot()
+
+        return self.appear_then_click(CONVT2_FINISH, offset=(20, 20), interval=1)
+    
+    # checking if we are actually in gear page
+    def is_in_gear(self):
+        interval= Timer (3)
+        interval.reached
+        return AUGMENT_INGEAR_CONFIRM.appear_on(self.device.image) and SHIP_INFO_EQUIPMENT_CHECK.appear_on(self.device.image)
+
+    # closing augment popup
+    def augment_popup_close(self, skip_first_screenshot=True):
+        logger.info('Augment popup close')
+        #self.interval_clear(AUGMENT_CANCEL)
+        while 1:
+            if skip_first_screenshot:
+                skip_first_screenshot = False
+            else:
+                self.device.screenshot()
+
+            if self.is_in_gear():
+                return
+            if self.appear_then_click(AUGMENT_CANCEL):
+                continue
+            if self.appear_then_click(AUGMENT_CANCEL2):
+                continue
+
+            # if self.handle_convt2_finish():
+            #     continue
+
+    #get out of gear or dock to main page
+    def augment_exit(self, skip_first_screenshot=True):
+        """
+        Pages:
+            in: is_in_gear or dock
+            out: MAIN
+        """
+        logger.info('Gear exit')
+        interval = Timer(3)
+        while 1:
+            if skip_first_screenshot:
+                skip_first_screenshot = False
+            else:
+                self.device.screenshot()
+
+            if self.ui_page_appear(page_dock):
+                logger.info(f'Gear exit at {page_dock}')
+                continue
+            # if self.appear_then_click(AUGMENT_CANCEL, interval=3, similarity=0.55):
+            # if self.augment_popup_close():
+            #     continue
+            # if self.handle_convt2_finish():
+            #     continue            
+            if interval.reached() and self.is_in_gear():
+                logger.info(f'is_in_gear -> {BACK_ARROW}')
+                self.device.click(BACK_ARROW)
+                self.device.click(BACK_ARROW)
+                interval.reset()
+                continue
+            if self.is_in_main(interval=5):
+                break
+    #one cycle of augment enhancing
+    def conv_uniq_enhance(self, skip_first_screenshot=False):
+        
+        logger.hr('Enhance unique module', level=2)
+        while not self.appear(AUGMENT_EMPTY):
+
+            if skip_first_screenshot:
+                skip_first_screenshot = False
+            else:
+                self.device.screenshot()
+            self.not_appear_then_click(AUGMENT_EMPTY, AUGMENT_INGEAR_CONFIRM, interval=3)
+
+            if skip_first_screenshot:
+                 skip_first_screenshot = False
+            else:
+                 self.device.screenshot()
```

Comment:
1. Your code is filled with click-sleep thingy, operation success is not guaranteed
see https://github.com/LmeSzinc/AzurLaneAutoScript/wiki/4.4.-State-loop